dashboard/src/shared/specs/mountWrapper.tsx

```diff
@@ -7,15 +7,17 @@ import thunk from "redux-thunk";
 
 import { IStoreState } from "../../shared/types";
 
-const mockStore = configureMockStore([thunk]);
+export const mockStore = configureMockStore([thunk]);
 
-const initialState = {
+export const initialState = {
```

Comment:
I don't think we need these extra exports, if we use `getStore(objectToMergeWithInitialState)` that I landed earlier.  See what you think.